### PR TITLE
create statics-pages and test views

### PR DESCRIPTION
**Pull request by tuansaker1412**
------------------------------------------
[o] Build Success!
------------------------------------------

[o] brakeman: success
[o] rspec: success
[o] eslint: success
[o] scss-lint: success
[o] bundle-audit: success
[o] reek: success
[o] rubocop: success
[o] rails_best_practices: success

### CODE REVIEW

**Comment by tuansaker1412** [comment]
ready

**Comment by dieunb** [comment]
commented

**Comment by tuansaker1412** [comment]
ready

**Comment by dieunb** [comment]
commented

**Comment by tuansaker1412** [comment]
ready

File: app/views/static_pages/index.html.erb [hunk 1]
```diff
@@ -0,0 +1,64 @@
+<%= render "layouts/modal" %>
+<div class="container">
+  <div class="row">
+    <div class="col s12">
+      <div class="index center">
+        <h3><%= t ".welcome" %></h3>
+        <h5><%= t ".get_start" %></h5>
+        <a class="waves-effect waves-light btn deep-purple" href="#modal1">
+          <%= t ".sign_in" %>
+          <i class="material-icons left">verified_user</i>
```
Comment (huongnguyenmta) [inline app/views/static_pages/index.html.erb]: I18n

File: app/views/static_pages/index.html.erb [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+        </a>
+        <a class="waves-effect waves-light btn black" href="#modal2">
+          <%= t ".sign_up" %>
+          <i class="material-icons left">supervisor_account</i>
```
Comment (huongnguyenmta) [inline app/views/static_pages/index.html.erb]: I18n

File: config/locales/en.yml [hunk 3]
```diff
@@ -1,23 +1,47 @@
...
+      welcome: "Welcome MLOG"
+      about: "ABOUT"
+    contact:
+      welcome: "Welcome MLOG"
+      contact: "CONTACT"
+    home:
+      welcome: "Welcome MLOG"
+      home: "HOME"
+
+
```
Comment (huongnguyenmta) [inline config/locales/en.yml]: remove empty line from 45-46

File: spec/controllers/static_pages_controller_spec.rb [hunk 4]
```diff
@@ -0,0 +1,33 @@
+require 'rails_helper'
```
Comment (huongnguyenmta) [inline spec/controllers/static_pages_controller_spec.rb]: using double quote

File: app/views/layouts/_header.html.erb [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+      <li><i class="material-icons"><%= t ".search" %></i></li>
+      <li class="input-search">
+        <input placeholder="<%= t ".search_placeholder" %>" id="search" type="text">
+      </li>
+    </ul>
+    <%= link_to t(".mlog"), root_path, class: "brand-logo center" %>
+    <ul id="nav-mobile" class="right hide-on-med-and-down">
+      <li><%= link_to t(".about"), about_path %></li>
+      <li><%= link_to t(".contact"), contact_path %></li>
+      <li><%= link_to t(".social"), home_path %></li>
```
Comment (huongnguyenmta) [inline app/views/layouts/_header.html.erb:13]: Click vào link này em vẫn chạy bình thường chứ?

**Comment by huongnguyenmta** [comment]
commented

File: app/views/layouts/_header.html.erb (hunk 5 above)
Comment (tuansaker1412) [reply app/views/layouts/_header.html.erb:13]: vâng ạ ._.

**Comment by tuansaker1412** [comment]
ready

**Pull request merged**